packages/ag-charts-community/src/chart/series/topology/geoGeometry.ts

```diff
@@ -0,0 +1,80 @@
+import { Path, ScenePathChangeDetection } from '../../../scene/shape/path';
+import type { Geometry, Position } from './geoJson';
+import { latY, lonX } from './mapUtil';
+
+export class GeoGeometry extends Path {
+    @ScenePathChangeDetection()
+    geometry: Geometry | undefined = undefined;
+
+    @ScenePathChangeDetection()
+    scale: number = 1;
+
+    override updatePath(): void {
+        this.path.clear();
```

Comment:
Nit: Use `trackChanges: true` to allow change detection on the shape?

Reply:
Discussed offline - the geometry is immutable, so the decorator on the `geometry` field should be enough to determine if the path is dirty.